ct-aa/dotTheory---TypeScript
# JS-974: Added React & TypeScript sample app

# Related Tickets
[JS-974](https://codeandtheory.atlassian.net/browse/JS-974)

# Description
- Added the sample app for the React and TypeScript training.

# Validation
- Open the terminal.
- Run cd "react-and-typescript".
- Run "npm install".
- Run "npm start".
- Open the browser and check the local host.
- The browser should display the Todo App and the User should be able to add an item using the input field and button.

https://github.com/ct-aa/dotTheory---TypeScript/assets/93116252/3a03d064-a742-4663-8c97-75dc65beac5d

# Risks
N/A
